cmd/agent/app/httpserver/srv.go

```diff
@@ -39,8 +40,9 @@ func NewHTTPServer(hostPort string, manager configmanager.ClientConfigManager, m
 	handler.RegisterRoutes(r)
 	errorLog, _ := zap.NewStdLogAt(logger, zapcore.ErrorLevel)
 	return &http.Server{
-		Addr:     hostPort,
-		Handler:  r,
-		ErrorLog: errorLog,
+		Addr:              hostPort,
+		Handler:           r,
+		ErrorLog:          errorLog,
+		ReadHeaderTimeout: 2 * time.Second,
```

Comment:
why is this necessary? Doesn't default work?

Reply:
Hi Yuri, without it linter was failing with this error ```G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)``` .